### PR TITLE
Support sending and receiving redundant audio data to help reduce the effects of packet loss on audio quality.

### PR DESCRIPTION
**Pull request by pracheth**
## ℹ️ Description
*provide a summary of the changes and the related issue, relevant motivation and context*
Support sending and receiving redundant audio data to help reduce the effects of packet loss on audio quality.

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update
    - [X] README update
    - [X] CHANGELOG update
    - [ ] guides update
- [X] This change requires a dependency update
    - [X] Amazon Chime SDK Media
    - [ ] Other (update corresponding legal documents)

## 🧪 How Has This Been Tested?
*describe the tests that you ran to verify your changes, any relevant details for your test configuration*
Dev testing and QA testing. Verified that audio sent and received is clear and audible with redundant audio enabled and disabled.

### Additional Manual Test
- [ ] Pause and resume remote video
- [ ] Switch local camera
- [ ] Rotate screen back and forth

## 📱 Screenshots, if available
*provide screenshots/video record if there's a UI change in demo app*
Will upload once available. There is a new picker on the join screen to allow enabling and disabling audio redundancy.

#### Integration validation (required before release)

- [X] Unit tests pass
- [ ] Build SDK against simulator with release options
- [ ] Build SDK against device with release options
- [ ] Build SDK against simulator with debug options
- [X] Build SDK against device with debug options
- [ ] Test Demo against simulator with SDK (debug build) in workspace
- [X] Test Demo against device with SDK (debug build) in workspace
- [ ] Test DemoObjC against simulator with SDK (debug build) in workspace
- [ ] Test DemoObjC against device with SDK (debug build) in workspace
- [ ] Test Demo against simulator with SDK.framework (release build)
- [ ] Test Demo against device with SDK.framework (release build)
- [ ] Test DemoObjC against simulator with SDK.framework (release build)
- [ ] Test DemoObjC against device with SDK.framework (release build)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
 ## Unreleased
 
+### Added
```
Comment (dylonChime) [inline CHANGELOG.md:3]: combine with Added section below

Comment (pracheth) [reply CHANGELOG.md]: Updated

**Review by dinmin-amzn — APPROVED** [review]

**Review by hokyungh — APPROVED** [review]

File: AmazonChimeSDK/AmazonChimeSDK/audiovideo/AudioVideoConfiguration.swift [hunk 2]
```diff
@@ -12,21 +12,31 @@ import Foundation
...
+        self.init(audioMode: .stereo48K, callKitEnabled: callKitEnabled, enableAudioRedundancy: true)
     }
 
-    public init(audioMode: AudioMode, callKitEnabled: Bool) {
+    convenience public init(enableAudioRedundancy: Bool) {
+         self.init(audioMode: .stereo48K, callKitEnabled: false, enableAudioRedundancy: enableAudioRedundancy)
+    }
+
+    convenience public init(audioMode: AudioMode, callKitEnabled: Bool) {
+         self.init(audioMode: audioMode, callKitEnabled: callKitEnabled, enableAudioRedundancy: true)
```
Comment (dylonChime) [inline AmazonChimeSDK/AmazonChimeSDK/audiovideo/AudioVideoConfiguration.swift:34]: This results in a behavior change for existing clients. Is that intentional? Should the default for `enableAudioRedundancy` be `false` here?

Comment (pracheth) [reply AmazonChimeSDK/AmazonChimeSDK/audiovideo/AudioVideoConfiguration.swift:34]: We want redundancy to be enabled by default so this should be true. This does have a dependency on media SDK though as the logic for redundancy resides in the media SDK.

**Review by dylonChime — APPROVED** [review]

**Pull request merged**